### ISSUE TITLE
md-select does not display dropdown even though DOM elements are present; causes page to freeze

### ISSUE THREAD

**Issue by sweatherall** [angular/material#7863]
I recently upgraded to Angular Material version 1.0.6 and my md-selects no longer work.

The md-option DOM elements are present, but when I click on the dropdown nothing appears and the page actually partially freezes up. There is no console error thrown. When I switch from md-select/md-option to a regular select/option the select dropdown works.

Here is some sample code I am using:

```
md-input-container(ng-if="bookable.seatsperbooking>1")

  md-select(
    aria-label='select number of seats'
    ng-model="appt.seats")

    md-option(
      ng-repeat='num in getnumber(seatsallowed) track by $index'
      value='{{num}}') {{ num }}
```

and here is a screenshot of the inspector, which shows the options list is present:
<img width="692" alt="screen shot 2016-04-03 at 5 43 35 pm" src="https://cloud.githubusercontent.com/assets/4127588/14235147/b5734236-f9c3-11e5-928d-de94209930fa.png">

I have tried variations (i.e. using ng-options within the md-select, using just 'option' within md-select) but nothing seems to work except completely replacing both md-select and md-option with select/option. But then the nice material styles are gone :[

**Comment by smolyakoff** [comment]
+1 Suffering from the same issue

**Comment by sweatherall** [comment]
@smolyakoff sorry I did not update this post, I got my issue resolved!

The issue was due to version incompatibility between angular and angular material.

I had just updated to angular-material 1.0.7, and left my angular version at 1.3.20.
Once I updated to angular 1.4 (or above) this was no longer an issue.

So it seems that md-select on angular material 1.0.7  is not compatible with angular 1.3.2 (and possibly other 1.3 versions, but I did not test). 
I can confirm that it works with angular 1.4.* and 1.5.*

I hope this helps!
